packages/fether-electron/src/main/index.js

```diff
@@ -66,7 +95,104 @@ app.on('will-quit', killParity);
 
 app.on('quit', () => {
   pino.info('Leaving Fether');
+  app.releaseSingleInstanceLock();
   killParity();
 });
 
+/**
+ * Security. Intercept new-window events (i.e. `window.open`) by
+ * overriding event.newGuest without using the supplied options tag
+ * to try to mitigate risk of an exploit re-enabling node integration
+ * despite being turned off in the configuration
+ * (i.e. `nodeIntegration: false`).
+ *
+ * References:
+ * - https://www.electronjs.org/blog/webview-fix
+ * - https://blog.scottlogic.com/2016/03/09/As-It-Stands-Electron-Security.html
+ */
+app.on('web-contents-created', (event, win) => {
+  win.on(
+    'new-window',
+    (event, newURL, frameName, disposition, options, additionalFeatures) => {
```

Comment:
This whole block should be removed, as it's exactly the same as  https://github.com/paritytech/fether/pull/451/files#diff-78e33ca3b76427b22991d49db58160e1R194 with different variable names

Reply:
Thanks, I've removed the duplication